vllm/v1/worker/tpu_model_runner.py

```diff
@@ -10,8 +10,14 @@
 import torch.distributed
 import torch.nn as nn
 # TPU XLA related
-import torch_xla.core.xla_model as xm
-import torch_xla.runtime as xr
+import os
+if os.environ.get("VLLM_TORCHAX_ENABLED", "0") == "1":
+    import torchax
+    env = torchax.enable_globally()
```

Comment:
do we have to run enable_globally() in multiple places or can we just do it once for the whole lifetime of the server?